preference/kubernetes/Deployment-v2.remote.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: preference
+    version: v2
+  name: preference-v2
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: preference
+      version: v2
+  template:
+    metadata:
+      labels:
+        app: preference
+        version: v2
+    spec:
+      containers:
+      - env:
+        - name: JAVA_OPTIONS
+          value: -Xms128m -Xmx256m -Djava.net.preferIPv4Stack=true -Djava.security.egd=file:///dev/./urandom
+        image: quay.io/rhdevelopers/istio-tutorial-preference:v2
```

Comment:
Same as before I think that the best thing would be using something like `...preference:1.0.0-v2` and so on, basically I am thinking on having a way to rollback to any version in case of errors.

Reply:
TBH i don't know where v2 is used , i  just added a new file in place existing one

Reply:
as a matter of fact i dont have v2 image locally.  going by semantic versioning `1.0.0-v2` is not a  good versioning :)

Reply:
It is used in service (pod) virtualization example.

Reply:
i dont have image with me, i can push it later with v2 style for easier mapping

Reply:
I would prever having something like v2, v2-virtualized, v2-????

Reply:
@lordofthejars can you please push the virtualised image to the quay.io repo so that we can use it ? shall we go with use the version as `v2` just to go with how we do in other part of the tutorial